rules.ini

```
@@ -227,6 +227,7 @@ UnityEntities[] = (?:^|/)StreamingAssets/SubScenes/
 UnityHDRP = (?:^|/)Unity\.RenderPipelines\.HighDefinition\.(?:Config\.Runtime|Runtime)\.dll$
 UnityHybridRenderer = (?:^|/)Unity\.Rendering\.Hybrid\.dll$
 UnityURP = (?:^|/)Unity\.RenderPipelines\.Universal\.Runtime\.dll$
+V8 = (?:^|/)v8(?:_libbase|_libplatform)?\.dll$
```

```suggestion
V8 = (?:^|/)v8\.dll$
```
Unless there's an example where v8.dll is not there, there's no use for that